roles/gpu_operator_wait_deployment/files/fetch_pod_metrics.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash -e
+
+if [[ "${1:-}" == "" || "${1}" == "-h" || "${1}" == "--help" ]]; then
+    cat <<EOF
+Fetch metrics from a Pod
+
+Usage: $0 <POD_PORT> <LABEL_SELECTOR> <NAMESPACE> <PROM_GREP_KEY>
+Example:
+  $0 9400 app=nvidia-dcgm-exporter gpu-operator-resources '#HELP'
+EOF
+    exit 0
+fi
+
+POD_PORT="$1"
```

Comment:
Nit: use
```bash
...=$1
shift
...=$1
shift
```
instead

It's easier to maintain - causes less diff / mistakes when modifying parameters

Reply:
ack, I'm not 100% convinced for "less mistakes", but perhaps